MetadataMigration/src/test/java/org/opensearch/migrations/EndToEndTest.java

```diff
@@ -141,13 +141,16 @@ private void metadataCommandOnClusters(TransferMedium medium,
         }
         arguments.metadataTransformationParams.multiTypeResolutionBehavior = MultiTypeResolutionBehavior.UNION;
 
-        // Set up data filters
-        var dataFilterArgs = new DataFilterArgs();
-        dataFilterArgs.indexAllowlist = Stream.concat(testData.blogIndexNames.stream(),
-            Stream.of(testData.movieIndexName, testData.indexThatAlreadyExists)).collect(Collectors.toList());
-        dataFilterArgs.componentTemplateAllowlist = testData.componentTemplateNames;
-        dataFilterArgs.indexTemplateAllowlist = testData.templateNames;
-        arguments.dataFilterArgs = dataFilterArgs;
+        // Set up data filters for ES 7.17 as we do not currently have transformations in place to support breaking
+        // changes from default templates and settings here
```

Comment:
Rather than keep the one off, could we fix the issue its running into?  If they seems like considerable work, could we create a jira or github issue to track what is going wrong without these settings?

Reply:
There was what seemed like some more involved decisions that should be made here so I have created a JIRA issue to track and updated the comment to link: https://opensearch.atlassian.net/browse/MIGRATIONS-2447